.github/workflows/ispc-ci.yml

```diff
@@ -662,12 +662,18 @@ jobs:
 
     - name: Build examples
       run: |
-        mkdir examples-build
         cmake -DCMAKE_C_FLAGS="/WX" -DCMAKE_CXX_FLAGS="/WX" examples-package\examples\cpu -B examples-build -Thost=x64 -G "Visual Studio 17 2022"
         cmake --build examples-build --target ALL_BUILD --config Release
       shell: cmd
 
-  linux-build-examples:
+    - name: Run examples
+      shell: cmd
+      run: |
+        # Rebuild with Ninja to avoid strange issue when msbuild reports error for gmres
+        cmake examples-package\examples\cpu -B examples-build-run -G Ninja
+        cmake --build examples-build-run --target run-cpu-examples --config Release
```

Comment:
You can just run `ctest` command directly and it solves the issue in CI, like here: https://github.com/ispc/ispc/commit/75b1445376b42859c73148c34c8b1048b5f1280a
Also it appears that rebuild with ninja is not needed when ctest is used:  https://github.com/ispc/ispc/actions/runs/17866472232